infra/cifuzz/run_fuzzers_test.py

```diff
@@ -263,7 +264,7 @@ def test_run_fuzz_targets_quits(self, mocked_create_fuzz_target_obj,
     magic_mock.target_name = 'target1'
     mocked_create_fuzz_target_obj.return_value = magic_mock
     self.assertTrue(runner.run_fuzz_targets())
-    self.assertIn('target1-testcase', os.listdir(runner.artifacts_dir))
+    self.assertIn('target1-address-testcase', os.listdir(runner.artifacts_dir))
```

Comment:
do we we want address or asan here, can add some mappings map.

Reply:
Don't think it's worth the complexity.